import-js.py

```diff
@@ -9,21 +9,24 @@
 
 
 def extract_path():
-    # We have to delimit the PATH output with markers because
-    # text might be output during shell startup.
-    out = subprocess.Popen(
-        [os.environ['SHELL'], '-l', '-c',
-            'echo "__SUBL_PATH__${PATH}__SUBL_PATH__"'],
-        env=os.environ,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE
-    ).communicate()[0].decode()
-    path = out.split('__SUBL_PATH__', 2)
-
-    if len(path) > 1:
-        return path[1]
-
-    return ''
+    if 'SHELL' in os.environ:
```

Comment:
Does sublime linter have this issue? How do they deal with it? If they don't, perhaps you could submit a PR to them as well?